content/en/synthetics/private_locations/dimensioning.md

```diff
@@ -62,6 +70,12 @@ When multiple workers are associated with a private location, each worker reques
 
 For example, ten tests are scheduled to run simultaneously on a private location that has two workers running. If Worker 1 is running two tests, Worker 1 can request three additional tests to run. If Worker 2 is not running any tests, Worker 2 can request the five following tests. The remaining two tests can be requested by whichever worker has finished running its test first (any worker that has available slots).
 
+### Troubleshooting
```

Comment:
How do you feel about adding this section into the Troubleshooting page (https://docs-staging.datadoghq.com/beth.glenfield/update-synthetics-docs/synthetics/troubleshooting/)?

Reply:
Let's do it! I wasn't sure where was appropriate 😄